disnake/enums.py

```diff
@@ -382,6 +383,11 @@ class AuditLogAction(Enum):
     automod_block_message                 = 143
     automod_send_alert_message            = 144
     automod_timeout                       = 145
+    onboarding_prompt_create              = 163
+    onboarding_prompt_update              = 164
+    onboarding_prompt_delete              = 165
+    onboarding_create                     = 166
+    onboarding_update                     = 167
```

Comment:
> These aren't documented, not sure if we want to implement them yet

I should definitely remove that if it's not documented, but I just didn't want to remove it in case it is documented.

Reply:
closing since audit logs were not added in api-docs pr. 2b81f4c90af51506ae7f6c0843f514b58be80f7a